src/app/component-wrapper/src/app/iq-select2/iq-select2.component.ts

```diff
@@ -93,7 +99,7 @@ export class IqSelect2Component implements AfterViewInit, ControlValueAccessor {
     private fetchAndfilterLocalData(term: string): Observable<IqSelect2Item[]> {
         if (!this.fullDataList) {
             return this.fetchData('')
-                .flatMap((items) => {
+                .mergeMap((items) => {
```

Comment:
shouldn't we use pipe(mergeMap(...)) since version 5.5? https://github.com/ReactiveX/rxjs/blob/master/doc/lettable-operators.md

Reply:
Both of them works I guess, maybe for backward compatibility. But it should be nice to replace them.

Reply:
As far as I know, the bundle size will be smaller when using pipe. Without using pipe the tree shaking doesn't know which rxjs operators are unused, so they wont be removed.

Edit: I accidentially clicked "start a review :/"